doc/source/serve/production.md

```diff
@@ -1,66 +1,63 @@
 :language: python
 :start-after: __fruit_example_begin__
 :end-before: __fruit_example_end__
 ```
 
-This graph is located in the `fruit.py` file and stored in the `deployment_graph` variable. It takes in requests containing a list of two values: a fruit name and an amount. It returns the total price for the batch of fruits.
-
-To run the deployment graph, we first navigate to the same directory containing the `fruit.py` file and then run `serve run fruit.deployment_graph`. `fruit.deployment_graph` is the deployment graph's import path (assuming we are running `serve run` in the same directory as `fruit.py`).
+This code is saved in the `fruit.py` file and the final deployment graph that we want to deploy is assigned to the `deployment_graph` variable.
+To run and test the application locally, use the `serve run fruit:deployment_graph` command:
 
 ```console
-# Terminal Window 1
+# Terminal window 1:
 
 $ ls
 fruit.py
 
-$ serve run fruit.deployment_graph
-2022-06-21 13:07:01,966  INFO scripts.py:253 -- Deploying from import path: "fruit.deployment_graph".
+$ serve run fruit:deployment_graph
+2022-06-21 13:07:01,966  INFO scripts.py:253 -- Deploying from import path: "fruit:deployment_graph".
 2022-06-21 13:07:03,774  INFO services.py:1477 -- View the Ray dashboard at http://127.0.0.1:8265
 ...
 2022-06-21 13:07:08,076  SUCC scripts.py:266 -- Deployed successfully.
+
 ```
 
 We can test this graph by opening a new terminal window and making requests with Python's [requests](https://requests.readthedocs.io/en/latest/) library.
 
 ```console
-# Terminal Window 2
-
-$ python3
+# Terminal window 2:
 
->>> import requests
->>> requests.post("http://localhost:8000/", json=["PEAR", 2]).json()
-    8
+$ curl -H "Content-Type: application/json" -d '{"PEAR": 2}' "http://localhost:8000/"
 ```
 
-Once we're finished, we can close the Python interpreter by running `quit()` and terminate the Ray cluster by typing `ctrl-C` int the terminal running `serve run`. This will tear down the deployments and then the cluster.
-
 (serve-in-production-config-file)=
 
-## Creating Your Serve Config File with `serve build`
+## Serve Config Files (`serve build`)
 
-You can create a Serve config file to manage your deployment graphs' configurations in production. The Serve CLI can "deploy" this file, using the [serve deploy](serve-in-production-deploying) command. This will deploy or update your deployment graphs in production. The file is written in YAML and has the following format:
+The most minimal config file for a Serve application consists of just an `import_path` (the same path you would pass to `serve run` when testing locally).
+This config file can be used with the [serve deploy](serve-in-production-deploying) command CLI or embedded in a [RayService] custom resource in Kubernetes to deploy and update your application in production.
+The file is written in YAML and has the following format:
 
 ```yaml
```

Comment:
Can we remove `host` and `port` from this yaml example? They're not included in the 2.0 release.